mogc_partnerships/patches.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+def patch_skip_activation_email():
+    try:
+        from openedx.core.djangoapps.user_authn.views import register
+    except ImportError:
+        return
```

Comment:
If we've got the logger active, we may want to log some stuff here too. E.g. "Unable to patch" (warning?), "Successfully patched _skip_activation_email" (info? debug?)

Reply:
Thanks, this is a good idea. I've noted it as #54 so that we can move forward without loosing track of this suggestion.